src/styles/global.css

```
@@ -1000,6 +1000,314 @@ body.project-page {
+
+.blog-canvas-header h1 {
+  font-family: "Cormorant Garamond", Garamond, serif;
+  font-weight: 600;
+  font-size: clamp(2rem, 3.4vw, 3.4rem);
+  line-height: 1.04;
+  letter-spacing: -0.012em;
+  margin-bottom: 0.8rem;
+}
+
+.blog-canvas-header .project-deck {
+  font-family: "Inter", system-ui, sans-serif;
```

Stylelint (17.6.0)

[error] Unexpected quotes around "Inter" (font-family-name-quotes)

(font-family-name-quotes)